commons-rest-test/src/main/java/com/mule/connectors/commons/rest/test/assertion/raml/RamlResponseAdapter.java

```diff
@@ -16,23 +15,18 @@
 
     private final int status;
     private final Values values;
-    private final byte[] content;
+    private final String content;
     private final String contentType;
 
     public RamlResponseAdapter(Response response) {
-        try {
-            this.status = response.getStatus();
-            Map<String, String[]> headers = new HashMap<>();
-            for (Map.Entry<String, List<Object>> entry : response.getHeaders().entrySet()) {
-                headers.put(entry.getKey(), Lists.transform(entry.getValue(), Functions.toStringFunction()).toArray(new String[0]));
-            }
-            this.contentType = response.getMediaType().getType();
-            values = new Values(headers);
-            content = response.readEntity(String.class).getBytes("UTF-8");
-        } catch (UnsupportedEncodingException e) {
-            throw new ContentEncodingException(e);
+        this.status = response.getStatus();
+        Map<String, String[]> headers = new HashMap<>();
+        for (Map.Entry<String, List<Object>> entry : response.getHeaders().entrySet()) {
+            headers.put(entry.getKey(), Lists.transform(entry.getValue(), Functions.toStringFunction()).toArray(new String[0]));
         }
-
+        this.contentType = response.getMediaType().toString();
+        values = new Values(headers);
+        content = response.readEntity(String.class);
```

Comment:
Didn't you recently change this the other way around? I mean, I think you were already using ```response.readEntity(String.class)```, then replaced with ```response.readEntity(String.class).getBytes("UTF-8")```, and now you are changing it back.